pkg/varnishcluster/controller/varnishcluster_secret_test.go

```diff
@@ -20,7 +20,7 @@ import (
 
 var _ = Describe("the varnish secret", func() {
 	validBackendPort := intstr.FromInt(8080)
-	vcNamespace := "varnish-secret"
+	vcNamespace := "default" // TODO: how to create varnish-secret keyspace in mock?
```

Comment:
Can you provide more detail about what this TODO means?

Reply:
If the namespace doesn't exist, the test will fail now whereas it didn't matter with the older k8s client version. Wondering if we should look into making this work with the custom namespace name.

Reply:
Oh interesting, I don't think it's too difficult to created a mock namespace...